packages/dds/tree/src/feature-libraries/treeTextCursor.ts

```diff
@@ -51,19 +62,31 @@ import { fail } from "../util";
  */
 export class TextCursor implements ITreeCursor<SynchronousNavigationResult> {
```

Comment:
For me, the name TextCursor brings to mind a cursor used in a text string, not one used in a simple tree which can be serialized to nice JSON.

Reply:
Yes, I think it could use some better naming.